storybook/src/stories/person/list-item-component.stories.mdx

```diff
@@ -63,19 +63,36 @@ You can create your own [`<PersonProvider>`](?path=/docs/data-person-provider--p
 
 Name                      | Type                            | Description
 ------------------------- | -----------------------------   | -----------
-`resolve`                 | `PersonResolver`               | Person resolver for a provider. 
+`personResolver`          | `PersonResolver`                | Person resolver for a provider. 
 
 ### Methods for PersonListItem
 
 | Name                                   | Return Type                     | Description
 | -------------------------------------- | -----------------------------   | -----------------------------
-| `getListItemDetailsByAzureId(azureId)` | `ListItemData*`                | Returns `ListItemData` for unique AuzureId.
-| `getListItemDetailsByUpn(upn)`         | `ListItemData*`                | Returns `ListItemData` for unique AuzureId.
+`getInfo()`                              | `PersonInfo`                    | Returns `PersonInfo` of person.
+`getPhoto()`                             | `PersonPicture**`                | Returns `PersonPicture` of person.
```

Comment:
Copy paste? Are these the correct methods?

Reply:
yes